app/services/approved_request_report.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+require "csv"
+
+class ApprovedRequestReport
+  attr_reader :start_date
+  attr_reader :end_date
+  attr_reader :file_path
+
+  def initialize(start_date:, end_date:, file_path: "./tmp/approved_request_report.csv")
+    @start_date = Date.strptime(start_date, "%m/%d/%Y")
+    @end_date = Date.strptime(end_date, "%m/%d/%Y")
+    @file_path = file_path
+  end
+
+  def csv
+    headers = ["start_date", "end_date", "event_name", "trip_id", "surname", "given_name", "department", "estimated_cost"]
+
+    CSV.open(file_path, "wb") do |csv|
+      csv << headers
+      TravelRequest.find_each do |request|
```

Comment:
As currently written, this loop runs 3 queries for every row in the CSV file.  Could we eager-load some of the joined values to cut down on the number of SQL queries?

```suggestion
      TravelRequest.includes(:estimates, creator: [:department]).find_each do |request|
```